src/utils/constants.ts

```diff
@@ -252,8 +252,10 @@ export const countriesList: string[] = countries
   .map((s) => s.name)
   .sort((a, b) => a.localeCompare(b));
 
+// encryption
 export const DACO_ENCRYPTION_ALGO = 'aes-128-cbc';
-export const CHAR_ENCODING = 'latin1';
+export const CHAR_ENCODING = 'hex';
+export const EMAIL_CONTENT_ENCODING = 'base64';
```

Comment:
can you please rename this to be more specific for encryption, like EXPORT_EMAIL_ENCRYPTION_ENCODING or something better just to not mix this var with other emails

Reply:
:+1: